src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+abstract class Figure implements AreaCalculator {
+    public static final int VALUE_LIMIT = 50;
+    protected String color;
+    protected double area;
```

Comment:
do we need area `field` here?
maybe we will be able to return our area just from the method